src/components/Button.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Text, View, StyleSheet, TouchableOpacity } from 'react-native';
+
+export type ButtonProps = {
+    title: string;
+    btnColor?: string;
+    onPress?: () => void;
+}
+
+const styles = StyleSheet.create({
+  button: {
+    height: 50,
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'center',
+    alignItems: 'center',
+    borderRadius: 8,
+    backgroundColor: '#000',
+    borderWidth: 2,
+    borderColor: '#000',
+    marginTop: 20,
+    marginLeft: 15,
+    marginRight: 15,
+  },
+  buttonLight: {
+    backgroundColor: '#fff',
+    borderWidth: 2,
+    borderColor: '#000'
+  },
+  text: {
+    color: '#000',
+    fontSize: 16
+  },
+  textLight: {
+    color: '#fff',
+    fontSize: 16
+  }
+});
```

Comment:
It's usually better to place styles `after` component, so when another developer (or event you) opens the file, he/she doesn't need to scroll in order to see main logic.